api/tests/test_archive_querier.py

```diff
@@ -528,3 +535,20 @@ def test_2x_max_results_in_one_bucket(table_maker, querier, record_maker):
     pages = get_all_pages(querier.query_by_time, [start, end, 'boo'])
     results = consolidate_pages(pages)
     assert len(results) == MAX_RESULTS * 2
+
+
+def test_latest_table_query(table_maker, querier, record_maker):
```

Comment:
I think you'll have to monkeypatch `DATALAKE_USE_LATEST_TABLE=true` so that the `HttpQuerier` ends up using the latest table. And just monkeypatching here might not work, since the fixture will already have been yielded. Maybe the move is to expand the `querier` fixture to have params for each combination of `(archive, http) * (use_latest, use_default)`?

Reply:
Added in commit [f19fad2](https://github.com/planetlabs/datalake/pull/89/commits/f19fad294d3d038c6b816cd2cb9b93d42acab725)